litmus-portal/frontend/src/views/Analytics/ApplicationDashboards/Table/TableData.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
same

Reply:
needed for the forEach loop

Reply:
This problem can be fixed, instead of using `?` wrap the forEach loop inside an if condition.